### PR TITLE
WS-652: structural changes, removal of submission form, added email form

### PR DESCRIPTION
**Pull request by emb03**
@z3cka glt would like this on stage today ASAP, since many people have to look at it before leaving for the day. He said it would be ok to leave off Captcha for now. Please review and merge to stage. Thank you!

### CODE REVIEW

**Comment by z3cka** [comment]
😃

**Comment by z3cka** [comment]
this looks mostly good. Let's separate the new form from the content stuff so we can get the content changes QAed and work on getting the form to send email as well. 

Also, lets try `published: false` on the old form rather than deleting it

File: pages/05.call-for-papers/form.md [hunk 1]
```diff
@@ -1,135 +0,0 @@
----
-title: Proposal Submission
```
Comment (z3cka) [inline pages/05.call-for-papers/form.md:2]: let's just add `published: false` here instead of deleting the form

**Comment by emb03** [comment]
@z3cka I put visible: false, published: false doesn't do anything.

**Comment by z3cka** [comment]
cleared Grav cache? _(I was afraid that it wouldn't work)_

**Comment by emb03** [comment]
yes, everytime

**Pull request closed without merging**